MIP0/mip0.md

```diff
@@ -7,287 +7,253 @@ Title: The Maker Improvement Proposal Framework
+-   **MIP Sets:** MIP Sets are groups of interdependent MIPs that work as a modular whole. See `MIP0c12`.
+-   **MIP Types:** MIPs can be either general, technical, or process MIPs. See `MIP0c5`.
+-   **Subproposals (SPs):** Subproposals are instances of Process MIP Components. See `MIP0c5`.  
+-   **Minimum Feedback Period:** The minimum amount of time within which the community can give feedback in response to a proposed MIP before it can advance to Formal Submission (see `MIP0c3`).  
+-   **Minimum Frozen Period:** The minimum amount of time during which a MIP must remain unchanged before it can advance to Formal Submission (see `MIP0c3`).
+-   **Governance Facilitator(s):** Governance Facilitators are tasked with ensuring the smooth operation of the governance process. This role must carry a wide range of activities: from general administration through signal gathering to governance scheduling. Governance Facilitators cannot delegate their formal responsibilities in the MIP Framework. Governance Facilitators have all the editorial powers a MIP Editor has and can bestow these powers on contributors.
+-  **MIP Editor(s):** MIP Editors enforce the administrative and editorial aspects of the overall MIPs process and program.
 
 ---
 ### MIP0c2: Core Principles
 
- 1. **Specificity:** A MIP needs to define and address a specific behaviour or single responsibility. MIPs with many different behaviours or responsibilities will not be allowed and must be split up into multiple MIPs.
-	 - This mitigates the risk of having “fine print” or potential attacks hidden in large, complex MIPs.
- 2. **Completeness:** A MIP or MIP Set is complete if it has all the necessary or appropriate parts that cover a whole behaviour and avoids being only a specific part of a greater whole.
-	 - This is important for both understandability, readability and accessibility of MIPs.
-3.  **Avoid overlap:** Multiple MIPs should not implement the same type of behaviour independently. For instance, there should not be two separate but interchangeable ways to do collateral onboarding.
-	- This way the primary and best-understood process for each particular behaviour will be fairly available to everyone, without risking having a knowledge gap that makes it possible for some actors with better access to information to use different and potentially better processes
+1. **Specificity:** A MIP needs to define and address a specific behavior or single responsibility. MIPs covering many different behaviors or responsibilities will not be allowed and must be split up into multiple proposals.
+
+2. **Completeness:** Be it a behavior or a responsibility, a MIP must not leave any essential aspect of its object undefined or unreferenced.
```

Comment:
Not sure this is better. Feels less clear to me. 'essential aspect of it's object' feels long-winded. Not sure what to suggest though.